ofl/gulzar/DESCRIPTION.en_us.html

```diff
@@ -11,5 +11,5 @@
     You can read more about the inspiration, design, and engineering of Gulzar at <a href="https://gulzarfont.org/">gulzarfont.org</a>.
 </p>
 <p>
-    To contribute, see <a href="https://github.com/simoncozens/Gulzar/">github.com/simoncozens/Gulzar/</a>.
+    To contribute, see <a href="https://github.com/googlefonts/Gulzar/">github.com/googlefonts/Gulzar/</a>.
```

Comment:
Normally I leave the final period on this sentence

Reply:
It seems it is there